setup.py

```diff
@@ -12,7 +12,7 @@
     packages=find_packages(where="src"),
     package_dir={"": "src"},
     package_data={
-        "isar_turtlebot": ["config/default.ini", "config/maps/turtleworld.json"]
+        "isar_turtlebot": ["settings/settings.env", "config/maps/turtleworld.json"]
```

Comment:
This file is empty and the readme is not updated with how it is used, right?

Reply:
Yes, this line should be removed. I will have a look at the readme.

Reply:
As for the readme there is no reference to the config that should have changed as far as I can tell.